html/html-spec.md

```diff
@@ -129,6 +129,30 @@ Would be rendered as:
   }
 ```
 
+### Escaping Content
```

Comment:
While you're editing this page, can you modify above section about `language` to note the default will be plaintext?

Reply:
I thought `qwiklabs-content-bundle-spec` is "git-whisperer agnostic", so that doesn't apply here - `<ql-code-block>` will actually automatically detect the language if it's not specified.

Reply:
Actually maybe not, I see in ql-variable i included some documentation on the markdown syntax. I should do something similar here for ql-code-block.

Reply:
Done, added a section about Markdown. Please review